front/src/applications/operationalStudies/components/MacroEditor/ngeOperations.ts

```diff
@@ -0,0 +1,54 @@
+import type { Distribution, TrainScheduleBase } from 'common/api/generatedEditoastApi';
```

Comment:
`osrdEditoastApi` should be used instead of `generatedEditoastApi`.

Reply:
Ref https://github.com/OpenRailAssociation/osrd/pull/8250

Reply:
Done